protocols/imap/src/main/java/org/apache/james/imap/processor/AbstractMailboxProcessor.java

```diff
@@ -102,7 +102,7 @@ protected final Mono<Void> doProcess(R acceptableMessage, Responder responder, I
                             return Mono.empty();
                         })
                         .onErrorResume(OverQuotaException.class, e -> {
-                            no(acceptableMessage, responder, HumanReadableText.FAILURE_OVERQUOTA);
+                            no(acceptableMessage, responder, HumanReadableText.FAILURE_OVERQUOTA, StatusResponse.ResponseCode.overQuota());
                             LOGGER.info("Processing failed due to quota restriction");
```

Comment:
Ideally we should use `ReactorUtils.logAsMono(() -> LOGGER.info(...))` syntax in order not to loose the MDC context